src/main/java/frc/robot/subsystems/swerve/SwerveSubsystem.java

```diff
@@ -409,6 +591,18 @@ public static Translation2d[] getModuleTranslations() {
     };
   }
 
+  public static VisionConstants[] getCameraConstants() {
+    return new VisionConstants[] {leftCamConstants, rightCamConstants};
+  }
+
+  /** Returns the module positions (turn angles and drive velocities) for all of the modules. */
+  @AutoLogOutput(key = "SwerveStates/Measured")
+  private SwerveModulePosition[] getModulePositions() {
+    SwerveModulePosition[] states = new SwerveModulePosition[4];
+    Arrays.stream(modules).map(Module::getPosition).toArray();
```

Comment:
This line doesn't do anything because it's not stored. Probably need to be assigned to the var above? Or just returned outright?